server/src/services/mail/index.js

```diff
@@ -0,0 +1,32 @@
+import nodemailer from "nodemailer";
+import { config } from "dotenv";
+config();
+
+const sendEmail = () =>{
+
+    let transporter = nodemailer.createTransport({
```

Comment:
```suggestion
    const transporter = nodemailer.createTransport({
```

Beside that, please move transporter declaration above sendEmail function because for now we initialize this variable each time when we are sending email, it would be better to initiliazie it once when app jus start